gui/app/main.js

```diff
@@ -421,7 +421,8 @@ app.on('ready', () => {
 app.on('window-all-closed', () => {})
 
 ipcMain.on('register-remote', (event, arg) => {
-  desktop.config.cozyUrl = arg.cozyUrl
+  const cozyUrl = desktop.checkCozyUrl(arg.cozyUrl)
+  desktop.config.cozyUrl = cozyUrl
```

Comment:
In the CLI, the user gives the URL and the path to sync, and then cozy-desktop opens his browser for OAuth, and when done, it persists all the config. Easy, CLI -> LIB -> CLI.

In the GUI, it is more complicated. First the user gives the URL, then cozy-desktop modifies the electron URL, then the user confirms, then cozy-desktop saves some informations, then the user gives the path, etc. It's a lot of context switching between the GUI and LIB, so there are some artifacts like this.